Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,81 @@
+/*Global Styles*/
+
+body{
+	background-color:#F6F6F6;
+	padding: 75;
```

Comment:
The browser is smart enough to assume you're talking about pixels, but don't rely on the browser for that, so make sure you always specify a unit, like `px in this case.